amsclient/amsclient.go

```diff
@@ -245,7 +245,7 @@ func (c *amsClientImpl) GetInternalOrgIDFromExternal(orgID types.OrgID) (
 	// 404 is used to ensure compatibility with Advisor UI, as it relies on 404 to render the correct response.
 	if len(orgIDs) == 0 {
 		err := errors.New(orgNoInternalID)
-		log.Error().Uint32(orgIDTag, uint32(orgID)).Err(err).Send()
+		log.Warn().Uint32(orgIDTag, uint32(orgID)).Err(err).Send()
```

Comment:
why are you changing this to Warn? this is a real error that should not happen. It's an edge case.

Reply:
After a good conversation in Slack, I am restoring this part of the PR and will be deciding about it later